### ISSUE TITLE
Add development instructions on README

### ISSUE THREAD

**Issue by emaiax** [wajahatkarim3/EasyValidation#29]
After a whole day trying to setup the project, I'm unable to even run the basic example tests without commenting the code or deleting .idea | .gradle folders.

<img width="1438" alt="screenshot 2018-11-01 00 36 20" src="https://user-images.githubusercontent.com/1289661/47831290-b6558880-dd6e-11e8-8042-36c8ec0537ae.png">

@wajahatkarim3 I'll try to figure it out what's going on and then I'll create development instructions, ok?

```
macOS 10.14 (18A391)
Kotlin [stable] @ 1.2.x
Android Studio, IntelliJ IDEA
JDK jdk1.8.0_172.jdk
```

**Comment by wajahatkarim3** [comment]
Fixed this issue in pull request #30 .
Now you can pull the `master` branch and it will compile successfully. 😄 
Thanks for your help and time 👍

**Comment by emaiax** [comment]
@wajahatkarim3 very nice! 🙏
